.github/workflows/build_win.bat

```diff
@@ -6,10 +6,10 @@ cmake . -G "NMake Makefiles" ^
     -DWITH_CPLEX=OFF ^
     -DWITH_GUROBI=OFF ^
     -DBUILD_CPP_TEST=OFF ^
-    -DCMAKE_PREFIX_PATH="%CONDA_PREFIX%" ^
-    -DCMAKE_INSTALL_PREFIX="%CONDA_PREFIX%" ^
-    -DPython_NumPy_INCLUDE_DIRS=$(python -c "import numpy; print(numpy.get_include())") ^
-    -DCMAKE_CXX_FLAGS="/std:c++17 /EHsc" ^
+    -DCMAKE_PREFIX_PATH:PATH="%CONDA_PREFIX%" ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%CONDA_PREFIX%" ^
+    -DPython_EXECUTABLE:PATH="%CONDA_PREFIX%\python.exe" ^
+    -DCMAKE_CXX_FLAGS="/EHsc" ^
```

Comment:
Why do you specify `/EHsc` i'm not familiar with windows but can you remove this?

Reply:
My windows knowledge is also limited... It seems like this is for suppressing some warnings during compilation. Do you think there is a problem in having it here?